agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileSender.java

```diff
@@ -92,8 +89,8 @@ public void run() {
         resultCode.join(30, TimeUnit.SECONDS); // wait max 30 seconds for request to be completed.
       }
     } catch (RuntimeException ex) {
-      logger.error(
-          "Unexpected error occurred while sending telemetries from the local storage.", ex);
+      LoggerFactory.getLogger(LocalFileSender.class)
```

Comment:
is there a need to inline here? it's conventional to have static final loggers

Reply:
it's only used here that's why i didn't create an instance var.

Reply:
static final is still (slightly) better so it doesn't need to get the logger each time it comes to this line (and also more conventional)